src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+abstract class Figure implements Drawable, GetArea {
+    private String color;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getFigureData() {
```

Comment:
The method `getFigureData` seems to only return the color of the figure. If the intention is to return more data about the figure, consider including all relevant information. If color is the only attribute, then this method may not be necessary as `getColor` already provides this information.